tests/unit/test_sql_query_chaining.py

```
@@ -714,3 +717,80 @@ def test_non_single_select_query_raises_exception(self):
+
+    def setUp(self):
+        sql_query_chaining._sql_query_registry.clear()
+        self._original_vars = vars(__main__).copy()
+
+    def tearDown(self):
+        sql_query_chaining._sql_query_registry.clear()
+        for key in list(vars(__main__).keys()):
+            if key not in self._original_vars:
+                delattr(__main__, key)
+
+    def _bind(self, name, value="bound"):
```

Ruff (0.15.14)

[warning] Missing return type annotation for private function `_bind`

Add return type annotation: `None`

(ANN202)